### ISSUE TITLE
[SUGGESTION] - Possibility to auto-add all DPS in the addon frame, EVEN in raid

### ISSUE THREAD

**Issue by youdontknowmu** [Xenknight61/EvokerAug#12]
Hi there,

I would like to make a suggestion on this wonderful addon !

At the moment the addon can auto Add DPS and Tanks oin MM mode (5 player group)
Is it possible to do the same thing with raid groups ? It's very long to auto add all DPS in the addon while raiding, especially with joining/leaving people

Thanks again !! :)

**Comment by Xenknight61** [comment]
Hello.

In the raid, we usually give buffs to the players we choose, we do not give it to all players, it seems more logical to add manual for this.
I realise that it is a bit difficult to add for Raid, I think how can I make it easier

thank you also for your nice suggestions and comments

**Comment by youdontknowmu** [comment]
Yep, usually in raid you roll between like 5-6 players to match their cd rotation 
But sometimes, those are assigned to specific roles or simply just die in the middle of the fight, thus making it usefully to have the others ''in case of'' or if you just did something wrong and are too far from all those guys.

We only raid with 13 dps but I thought many times that would have saved time, but only if I could effectively sort them once added

(Or maybe right-click on of the added dps, and have an option Like ''always show on top'' and all the dps that you choose to see on top stays on top of the others)

Thanks you for the answers ^^
